migrate/20251107_add_annotation_model.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:annotation) do
+      # UBID.to_base32_n("an") => 341
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_ubid_uuid(341)")
+      column :description, :text
```

Comment:
Might be useful to have a jsonb column or uuid[] column that can be used to reference related objects. These could be autolinked in the Admin UI.